nuvlaedge/agent/orchestrator/kubernetes.py

```diff
@@ -86,6 +89,25 @@ def _sanitize_k8s_object(cls, data):
         else:
             return data
 
+    @staticmethod
+    def _list_helm_releases():
+        cmd = 'helm list --all-namespaces -o json'
+        try:
+            result = run(cmd, capture_output=True,
+                         timeout=10, encoding='UTF-8',
+                         shell=True, check=False)
```

Comment:
`shell=True` is usually not recommended for [security reasons](https://docs.python.org/3/library/subprocess.html#security-considerations).
Here it's ok in term of security but it will most probably trigger an alert on SonarCloud when we will reactivate it.
Also it create an extra call to a shell, so it's less efficient.

Reply:
Agreed, `shell=True` is redundant here.